.github/workflows/ci.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        ruby: ["3.0", "3.1", "3.2", "head"]
+        ruby: ["3.0", "3.1", "3.2", "head", "jruby", "jruby-head"]
```

Comment:
I think just `jruby-head` is probably fine

Reply:
@vinistock I tried that too, But it was still failing with cryptic sorbet error. I haven't worked with sorbet before to debug that.